Add `execute`, `get_policy` and `set_policy`

pitkley opened the pull request:
While using this library I noticed that I lacked direct access to the iptables-instance and getting/setting the policy on a chain.

Functions I added:

* `execute`: a very thin wrapper around `run`, only specifying the table, leaving the rest to the user.

    Making `run` public might be an option, too.

* `get_policy`: get the policy for a given table and chain
* `set_policy`: set the policy for a given table and chain

I'm not sure about the names, or the implementations, just let me know if you have better ideas. 👍

yaa110 commented:
Thanks for your contribution, I like your idea about the names. Could you please add some tests (in [iptables_test.rs](https://github.com/yaa110/rust-iptables/blob/master/tests/iptables_test.rs) files) for those new functions?

pitkley commented:
Thanks for the response, I have added some tests!

yaa110 commented on tests/iptables_test.rs:
```diff
@@ -41,5 +44,52 @@ fn filter(ipt: iptables::IPTables, name: &str) {
+}
+
+#[test]
+fn test_set_policy() {
```
We are not allowed to change current settings on tests. We should ensure that the last line is executed  in all situations even if a command panics.

pitkley replied:
Very good point, I worried about that too. I have updated the test to hopefully reflect this requirement.

yaa110 replied:
thank you for the change, it is more reliable now. However, I think it is better to use [ignore attribute](https://doc.rust-lang.org/book/testing.html#the-ignore-attribute) for this test to be optional.

pitkley commented:
I have added the ignore attribute. I have also updated the Travis configuration to execute the ignored tests, let me know if you disagree with that.

Furthermore, I have added an additional function `chain_exists` which returns `true` if a chain exists in a table, `false` otherwise. I didn't want to split this one up into a separate PR, but can do so if you would like me to.

yaa110 commented:
Thank you

pitkley commented:
Thanks for merging and the guidance!

yaa110 commented:
@pitkley please add your name to authors section of cargo.toml
